examples/cassandra-lib/client-config.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 endpoints:
   - 127.0.0.1:9042
-op_timeout: 5000000000  # 5 seconds (creating a new keyspace is very slow on travis)
+op_timeout: 5000000000  # 5 seconds (creating a new keyspace is slow on travis)
```

Comment:
it is just "slow" now? :)

Reply:
This was done to shorten the line and avoid showing warning from inspection.. hehe.